core/templates/pages/contributor-dashboard-page/services/contribution-and-review.service.ts

```diff
@@ -170,16 +171,18 @@ export class ContributionAndReviewService {
   }
 
   async getUserCreatedQuestionSuggestionsAsync(
-      shouldResetOffset: boolean = true
+      shouldResetOffset: boolean = true,
+      sortKey: string = ''
   ): Promise<FetchSuggestionsResponse> {
+    this.userCreatedQuestionFetcher.sortKey = sortKey;
     return this.fetchSuggestionsAsync(
       this.userCreatedQuestionFetcher,
       shouldResetOffset);
   }
 
   async getReviewableQuestionSuggestionsAsync(
       shouldResetOffset: boolean = true,
-      sortKey?: string
+      sortKey: string = ''
```

Comment:
This should be `null` if you don't actually have a sort key, rather than `''`.

(Better to always have a sort key though, if possible.)